nearcore/src/metrics.rs

```diff
@@ -0,0 +1,28 @@
+use near_metrics::{try_create_histogram_vec, HistogramVec};
+use once_cell::sync::Lazy;
+
+pub static APPLY_CHUNK_DELAY: Lazy<HistogramVec> = Lazy::new(|| {
+    try_create_histogram_vec(
+        "near_apply_chunk_delay",
```

Comment:
This metric name should contain a unit in it.

Reply:
Done